README.md

```diff
@@ -1,48 +1,47 @@
 # intercom-php
 
-[![Code
-Climate](https://codeclimate.com/repos/537da4a7e30ba062b101be9c/badges/2aa25d4736f09f40282e/gpa.svg)](https://codeclimate.com/repos/537da4a7e30ba062b101be9c/feed) [![Circle CI](https://circleci.com/gh/intercom/intercom-php.png?style=badge)](https://circleci.com/gh/intercom/intercom-php)
+[![Code Climate](https://codeclimate.com/repos/537da4a7e30ba062b101be9c/badges/2aa25d4736f09f40282e/gpa.svg)](https://codeclimate.com/repos/537da4a7e30ba062b101be9c/feed) [![Circle CI](https://circleci.com/gh/intercom/intercom-php.png?style=badge)](https://circleci.com/gh/intercom/intercom-php)
 
 Official PHP bindings to the Intercom API
 
 ## Installation
 
-Requires PHP 5.6.
+Requires PHP 5.6 or above.
 
-The recommended way to install intercom-php is through [Composer](https://getcomposer.org):
-
-First, install Composer:
-
-```sh
-curl -sS https://getcomposer.org/installer | php
-```
+This library uses [HTTPPlug](https://github.com/php-http/httplug) as HTTP client. HTTPPlug is an abstraction that allows this library to support many different HTTP Clients. Therefore, you need to provide it with an adapter for the HTTP library you prefer. You can find the available adapters [here](http://docs.php-http.org/en/latest/clients.html). This documentation assumes you use the Guzzle6 Client, but you can replace it with any adapter that you prefer.
```

Comment:
Maybe you could add the following link too, due to is an easy way to see all packages that are including the HTTP Client implementation that allows HTTP Plug do its magic: https://packagist.org/providers/php-http/client-implementation